GraphBase/src/algorithm/Traversal.java

```diff
@@ -0,0 +1,73 @@
+package algorithm;
+
+import graph.Graph;
+import graph.Vertex;
+
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Queue;
+import java.util.LinkedList;
+import java.util.Stack;
+
+public class Traversal {
+
+  public ArrayList<Integer> BFS(Graph graph, Integer entryId) {
```

Comment:
Try use int instead. If you use Integer here, the user can input null, which is bad. There is a generaly rule, use simple type wherever it can (int (Integer), float (Float), long (Long) ...).